microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -29,44 +37,24 @@ class Dpdk(TestSuite):
              with one interface for management.
             More detailes refer https://docs.microsoft.com/en-us/azure/virtual-network/setup-dpdk#prerequisites # noqa: E501
         """,
+        priority=1,
         requirement=simple_requirement(
+            min_core_count=8,
```

Comment:
What's the reason to add this condition?

Reply:
DPDK requires at least 8 cores

Reply:
Ah! I'm glad you asked. I had it in my head that azure required 8 cores for DPDK w multiple nics. Now that I am double checking the requirements are more complex that I remember, so I'll fix that up and submit changes

Reply:
fixed, it's 4 for hyper-threaded hosts (need >one extra physical core for dpdk)

Reply:
Where is this requirement from? dpdk or Azure? If it's dpdk, it's ok to keep it here. If it's from Azure, it's better to convert the dpdk to a feature, so we can customize the requirement of the dpdk in Azure platform.

Reply:
dpdk

Reply:
Azure's requirements are bundled up into SRIOV support so I don't think we need to make dpdk it's own feature

Reply:
SRIOV has its requirement already. So it doesn't need the extra core count in test cases, if the dpdk has same requirement like SRIOV. The feature is used to manage requirement and logics, so people know where a requirement is from, and how it applies to different platforms. It helps to improve maintenance ability of code.

Reply:
If it's a requirement of dpdk, a reference can help track the requirement here. As we met problems in v2, it's painful to understand the logic and requirements, if the documentation is not enough.

Reply:
sounds good, removing it here